config/plugins/interactive_environments/jupyter/config/jupyter.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE interactive_environment SYSTEM "../../interactive_environments.dtd">
+<interactive_environment name="Jupyter (Programming Environment)">
```

Comment:
More people know what IPython is than Jupyter I would assume, also IPython has a more informative name. I'm fine either way but my vote would be for "Jupyter (Scripting Environment)".  :horse:.